pkg/engine/cmd.go

```diff
@@ -203,10 +204,9 @@ var ignoreENV = map[string]struct{}{
 }
 
 func appendEnv(envs []string, k, v string) []string {
-	for _, k := range []string{k, env.ToEnvLike(k)} {
-		if _, ignore := ignoreENV[k]; !ignore {
-			envs = append(envs, k+"="+v)
-		}
+	//fmt.Printf("%s=%s\n", k, v)
+	if _, ignore := ignoreENV[k]; !ignore {
+		envs = append(envs, strings.ToUpper(k)+"="+v)
```

Comment:
Do we want to use `strings.ToUpper(env.ToEnvLike(k))` here to get right of `-` and `.`?

Reply:
in my heart i hope that nobody is trying to create env vars like that, but you are right it would not hurt to do!